Add Startpage (#1928)
sindresorhus/awesome: readme.md

- [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
- [Credit Modeling](https://github.com/mourarthur/awesome-credit-modeling#readme) - Methods for classifying credit applicants into risk classes.
- [Ansible](https://github.com/KeyboardInterrupt/awesome-ansible#readme) - A Python-based, open source IT configuration management and automation platform.

> - [Startpage](https://github.com/jnmcfly/awesome-startpage#readme) - A startpage will be shown when you start your browser, new tab or window.

Reviewer: ```suggestion
- [Startpage](https://github.com/jnmcfly/awesome-startpage#readme) - The page that is shown when you open your browser, new tab or window.
```

- [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
- [Credit Modeling](https://github.com/mourarthur/awesome-credit-modeling#readme) - Methods for classifying credit applicants into risk classes.
- [Ansible](https://github.com/KeyboardInterrupt/awesome-ansible#readme) - A Python-based, open source IT configuration management and automation platform.

> - [Startpage](https://github.com/jnmcfly/awesome-startpage) - Startpages for more productivity and efficiency.

Reviewer: The description should explain what a "startpage" is.

Reviewer: You are missing `#readme`.

- [Biological Visualizations](https://github.com/keller-mark/awesome-biological-visualizations#readme) - Interactive visualization of biological data on the web.
- [Veganism](https://github.com/sdassow/awesome-veganism#readme) - Making the plant-based lifestyle easy and accessible.

> 

Reviewer: Don't do unrelated changes.
